repository/f.json

```diff
@@ -169,6 +169,17 @@
 				}
 			]
 		},
+		{
+			"name": "FallSyntaxColorScheme",
```

Comment:
I would use some spaces in that name, e.g. "Fall Color Scheme"?

Reply:
Yup, changed